.github/pull_request_template

```diff
@@ -15,7 +15,7 @@ Your changes will be ready for merging after each of the steps below have been c
 
 ### Testing
 - [ ] Unit tests written
-- [ ] Tested locally with Inframock
+- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
```

Comment:
if you are taking the time to fix the PR template, do you mind also taking this ticket on: https://biomage.atlassian.net/browse/BIOMAGE-596 and making the  "Anything else the reviewers should know about the changes here" and "Code Changes" fields clearer?

Reply:
I see you already merged the others, so no point doing this anymore